webpack/assets/javascripts/react_app/components/BreadcrumbBar/components/__snapshots__/Breadcrumb.test.js.snap

```diff
@@ -67,7 +77,12 @@ exports[`Breadcrumbs renders title override 1`] = `
     title="override title"
   >
      
-    override title
+    <span
```

Comment:
do we need this span when the string is below the max-length ?

Reply:
Not really, removed